draft-add-requirements.md

```diff
@@ -104,92 +108,137 @@ the IETF may publish, such as DNS-over-QUIC {{?I-D.ietf-dprive-dnsoquic}}.
+There are two possible sources of designation.
 
-Network-identified is preferred since it comes from the same source of information, and removes the need to talk to the Do53 resolver
-at all. However it cannot be the sole mechanism, at least for several years, since there is a large installed base of local
-network equipment that is difficult to upgrade with new features. Hence the second mechanism must support being able to
-announce an equivalent resolver using only existing widely-deployed DNS features.
+* The local network can designate one or more encrypted DNS resolvers (B, C, etc) in addition to any Do53 resolver (A) it may offer. This is known as network-identified.
+* During communication with the (often unencrypted) resolver (A), this resolver can designate one or more encrypted DNS resolvers (B, C, etc). This is known as resolver-identified.
+
+Network-identified has the advantages that it derives from the same source of information as the network's Do53 announcement, and
+removes the need to talk to the Do53 resolver at all. However it cannot be the sole mechanism, at least for several years, since
+there is a large installed base of local network equipment that is difficult to upgrade with new features. Hence the second mechanism
+should support being able to designate resolvers using only existing widely-deployed DNS features.
 
 ## Local addressing
 
 Many networks offer a Do53 resolver on an address that is not globally meaningful, e.g. {{?RFC1918}}, link-local
-or unique local addresses. To support the discovery of Encrypted DNS in these environments, a means is needed for
-the discovery process to work from a locally-addressed Do53 resolver to an Encrypted DNS resolver that is accessible
+or unique local addresses. To support the discovery of encrypted DNS in these environments, a means is needed for
+the discovery process to work from a locally-addressed Do53 resolver to an encrypted DNS resolver that is accessible
 either at the same (local) address, or at a different global address. Both options need to be supported.
 
-# Network-identified encrypted resolvers
+## Use of designation information
+
+After the client receives designation information, it must come to a decision on whether and when to use any of the
+designated resolvers. To support this process, it would be advantageous for a solution to enable the client to validate
+the source of the assertion in some way. For example for both network-identified and resolver-identified designation, it
+may be possible to verify that the designation comes from an entity who already has full control of the client's Do53
+queries. It would be beneficial to extend this to defend against attackers that have only transient control of such queries.
+
+Clients may also seek to validate the identity of the designated resolver, beyond what is required by the relevant
+protocol. In general, clients may impose arbitrary additional requirements and heuristics as they see fit.
+
```

Comment:
The additional requirements and heuristics used by the client is out of scope of this WG.

Reply:
I agree, but I read this as "implementors of protocols fulfilling this requirements doc should be aware that clients may do other checks" rather than "fulfillers of this document ought to do something themselves".

Reply:
Tommy's interpretation is what I had in mind. I'll clarify the text.